gaia-sdk-javascript/src/api/IdentityOp.ts

```diff
@@ -20,4 +38,56 @@ export class IdentityOp {
                 throw new Error('Exporting identity with id ' + identityId + ' failed: ' + reason);
             }));
     }
+
+    private uuidv4() {
```

Comment:
Koennen wir net UUID.randomUUID().toString() verwenden?
so wie in gaia-sdk-javascript/src/mqtt/QueueOptions.ts